flags/flags.go

```diff
@@ -54,14 +55,15 @@ func (flags *Flags) ConfigFilePath() string {
 
 // RenderIf displays special-case information based on the flags passed
 // in, if any flags were passed in
-func (flags *Flags) RenderIf(version, date string, config *config.Config) {
+func (flags *Flags) RenderIf(config *config.Config) {
 	if flags.HasModule() {
 		help.Display(flags.Module, config)
 		os.Exit(0)
 	}
 
 	if flags.HasVersion() {
-		fmt.Printf("%s (%s)\n", version, date)
+		info, _ := debug.ReadBuildInfo()
+		fmt.Println(info)
```

Comment:
I believe this dumps everything in `BuildInfo`, which isn't what we want the `flags.HasVersion()` command to do. 

The original version outputs a version string and compile date: 

```
❯ wtfutil --version
v0.41.0-102-gf3b582 (2022-08-27T18:14:58-0700)
```
which we still need.